src/components/NewSnippet.jsx

```diff
@@ -1,49 +1,87 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import { Controlled as CodeMirror } from 'react-codemirror2';
 
 import 'codemirror/lib/codemirror.css';
 
 import Title from './common/Title';
 import Input from './common/Input';
 import Syntaxes from './Syntaxes';
+import * as actions from '../actions';
 
 import '../styles/NewSnippet.styl';
 
 class NewSnippet extends React.Component {
-  constructor() {
-    super();
-    this.state = { code: '' };
+  constructor(props) {
+    super(props);
+    this.state = {
+      content: '',
+      title: '',
+      tags: [],
+      syntax: '', // eslint-disable-line react/no-unused-state
+    };
+    this.postSnippet = this.postSnippet.bind(this);
+    this.onSyntaxClick = this.onSyntaxClick.bind(this);
+    this.onInputChange = this.onInputChange.bind(this);
   }
+
+  onSyntaxClick(syntax) {
+    this.setState({ syntax }); // eslint-disable-line react/no-unused-state
```

Comment:
Why do we need this *ignore* rule in two places?

Reply:
because our lint checker creates to error messages for those two lines